content/chainguard/chainguard-images/working-with-images/security-advisories/how-chainguard-issues/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "How Chainguard Issues Security Advisories"
+linktitle: "Chainguard's Advisories"
+aliases: 
+type: "article"
+description: "The life cycle of Chainguard-Issued Security Advisories"
+date: 2024-07-26T18:09:12+00:00
+lastmod: 2024-07-30T14:59:20+00:00
+draft: false
+tags: ["Overview", "Product", "Chainguard Images", "CVE"]
+images: []
+menu:
+  docs:
+    parent: "security-advisories"
+weight: 020
+toc: true
+---
+
+When you scan a newly-built Chainguard Image with a vulnerability scanner, typically, no CVEs will be reported. However, as software packages age, more vulnerabilities are reported, and CVEs begin to accumulate in images. When this happens, Chainguard releases security advisories to communicate these vulnerabilities to downstream Images users.
+
+Chainguard publishes its security advisories to a dedicated [Security Advisories page](https://images.chainguard.dev/security/) on its Images Registry. Here, you can find a complete listing of CVEs found in various Chainguard Images, including their CVE ID, affected packages, and vulnerability status. Essentially, these security advisories act as metadata for a security vulnerability.
```

Comment:
```suggestion
Chainguard publishes its security advisories to a dedicated [Security Advisories page](https://images.chainguard.dev/security/) on its Images Directory. There, you can find a complete listing of CVEs found in various Chainguard Images, including their CVE ID, affected packages, and vulnerability status. Each advisory is built from the metadata of a given security vulnerability.
```

I lean on calling it the Directory, don't want to folks to confuse it with a regular registry. 

Also I thought the last sentence was a bit off, i tried to reframe it